plugins/nessus.rb

```diff
@@ -1065,11 +1075,17 @@ def cmd_nessus_db_scan(*args)
         end
         targets.chop!
         print_status("Creating scan from policy #{policy_id}, called \"#{name}\" and scanning all hosts in all the workspaces")
-        scan = @n.scan_create(policy_id, name, desc, targets)
+        et=Hash.new
+        et['enabled']=false
+        et['launch']='ONETIME'
+        et['name']=name
+        et['text_targets']=targets
+        et['description']=desc
+        et['launch_now']=true
+        scan = @n.scan_create(policy_id, et)
         if !scan["error"]
           scan = scan["scan"]
-          print_status("Scan ID #{scan['id']} successfully created")
-          print_status("Run nessus_scan_launch #{scan['id']} to launch the scan")
```

Comment:
Yes, if you create the scan with nessus_scan_new... and also if you create the scan from Nessus UI and want to launch it from the mf.